client/model.go

```diff
@@ -76,12 +76,12 @@ const (
 
 type TemplateRetryOn struct {
 	Times      int    `json:"times,omitempty"`
-	ErrorRegex string `json:"errorRegex,omitempty"`
+	ErrorRegex string `json:"errorRegex"`
```

Comment:
why remove the `omitempty`?
if the user didn't put any what we want to do?

Reply:
We must remove `omitempty` to enable proper delete of this field